src/chocolatey/infrastructure.app/utility/ArgumentsUtility.cs

```diff
@@ -79,11 +89,32 @@ public static IEnumerable<string> DecryptPackageArgumentsFile(IFileSystem fileSy
                 yield break;
             }
 
-            // The following code is borrowed from the Chocolatey codebase, should
-            // be extracted to a separate location in choco executable so we can re-use it.
-            var packageArgumentsUnencrypted = arguments.Contains(" --") && arguments.ToStringSafe().Length > 4
-                ? arguments
-                : NugetEncryptionUtility.DecryptString(arguments);
+            string packageArgumentsUnencrypted = string.Empty;
+
+            try
+            {
+                // The following code is borrowed from the Chocolatey codebase, should
+                // be extracted to a separate location in choco executable so we can re-use it.

```

Comment:
I'm not sure this comment is... relevant anymore?

Reply:
It was actually borrowed from elsewhere 😂 I've removed it.